src/Automation/Automation.Sdk/README.md

```diff
@@ -25,31 +25,32 @@ title: AutomationClient
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/preview/2020-01-13-preview/sourceControlSyncJobStreams.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/dscConfiguration.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/job.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/certificate.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/credential.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/jobSchedule.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/schedule.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/module.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/variable.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/connection.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/connectionType.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControl.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControlSyncJob.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControlSyncJobStreams.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/dscConfiguration.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/job.json
```

Comment:
Please don't use the swagger in your own repo.

Reply:
Sorry i have updated the reference from azure repo. Thanks for pointing out.